test/controllers/auctions_controller_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require 'date'
+
+class AuctionsControllerTest < ActionDispatch::IntegrationTest
+  test 'should get /auctions endpoint' do
+    get '/auctions'
+    assert_response :success
```

Comment:
I understand you're trying to separate the tests concerns, but in this case, Id personally include this assertion in the other test.

3 reasons:
 - This assertion is not that much disconnected to what the other test is doing
 - I find it useful to check for the status as a "pre-assertion" on my controller tests. That way, if there's some other problem like authentication, or I send the wrong id, the failure reason is more evident than trying to understand why my body couldn't be JSON parsed.
 - Rails controller tests (without any kind of mocking) are generally slow-ish since they have to go through the whole stack + DB,  so I try not to create very skinny tests.